todos-postgres-tutuorial/Sources/PostgresClientService.swift

```diff
@@ -0,0 +1,14 @@
+import Hummingbird
+@_spi(ConnectionPool) import PostgresNIO
```

Comment:
I like their connection pool, like everyone, but @_spi in a tutorial doesn't feel right

Reply:
Yes I know. But I don't have an alternative unless I use the Vapor alternative. Remember this is still alpha software.

Reply:
I'll mention something in the tutorial text for the moment and hope Fabian removes this eventually